samples/kotlin-android-app/src/main/java/com/segment/analytics/next/MyFirebaseService.kt

```diff
@@ -19,19 +17,8 @@ class MyFirebaseService : FirebaseMessagingService() {
 
     val TAG = "MyFirebaseService"
 
-    override fun onNewToken(s: String) {
-        Log.e(TAG, s)
-        FirebaseInstanceId.getInstance().instanceId
-            .addOnCompleteListener(OnCompleteListener { task ->
-                if (!task.isSuccessful) {
-                    Log.w(TAG, "getInstanceId failed", task.exception)
-                    return@OnCompleteListener
-                }
-
-                // Get new Instance ID token
-                val token = task.result.token
-                Log.e(TAG, token)
-            })
+    override fun onNewToken(token: String) {
+        Log.e(TAG, token)
```

Comment:
why change this?

Reply:
FirebaseInstanceId is deprecated. no longer needed to get the token. see here https://firebase.google.com/docs/cloud-messaging/android/client